templates/registration/login.html

```diff
@@ -2,22 +2,22 @@
 
```

Comment:
Make sure you've added a blank line at the end to all your files including `.html` files as per the checklist.